A_NEW_FILE.md

```diff
@@ -0,0 +1,3 @@
+This is a new file.
```

Comment:
This file is markdown file. I love markdown. [Check this out](https://guides.github.com/features/mastering-markdown/)